priv/update-swagger-url

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env escript
+
+-module('update-swagger-url').
+
+main(_) ->
+  %% Define the path to the template and the output file
+  TemplateFilePath = "priv/swagger/swagger-initializer.template",
+  OutputFilePath = "priv/swagger/swagger-initializer.js",
+
+  %% Read the contents of the template file
+  case file:read_file(TemplateFilePath) of
+      {ok, Content} ->
+          %% Get the configuration for the path from the application environment
+          Path = application:get_env(cowboy_swagger, path, "/api-docs"),
```

Comment:
This won't work. This script will run at compile-time in an erlang VM where `cowboy_swagger` is not yet compiled, and thus… not yet **loaded**. Therefore, no matter what the users of this library put on `path` in their `sys.config` files, it will be invisible to this script and therefore we'll end up with `"/api-docs"` everywhere. We need a different approach, one that acts at runtime instead of compile-time.